assets/js/payment-method-extensions/payment-methods/stripe/stripe-utils/type-defs.js

```diff
@@ -294,6 +294,8 @@
  *                                                          inline cc
  *                                                          form or separate inputs.
  * @property {{[k:string]:CreditCardIcon}} icons            Contains supported cc icons.
+ * @property {boolean}                     allowSavedCards  Used to indicate whether saved cards
+ *                                                          can be used.
```

Comment:
Not related to this pull but I discovered it when in the process of debugging so added the missing type-def property declaration.